.github/workflows/deploy-staging.yml

```diff
@@ -16,6 +16,8 @@ jobs:
   build_and_deploy:
     env:
       NUXT_UI_PRO_LICENSE: ${{ secrets.NUXT_UI_PRO_LICENSE }}
+      ALGOLIA_SEARCH_API_KEY: ${{ secrets.ALGOLIA_SEARCH_API_KEY }}
+      ALGOLIA_APPLICATION_ID: ${{ secrets.ALGOLIA_APPLICATION_ID }}
```

Comment:
Unsure if this is a valid concern but should we avoid using the same api key / id for both prod and staging environments. Reasoning is, and I may be mistaken here, indexing data into Algolia from both environments, using the same API key might cause the staging environment to overwrite or pollute the production indices. This could lead to incorrect search results in your production environment if, for instance, test data from staging gets indexed. 

Might not be a concern but just an initial thought. @itsacoyote ?

Reply:
This is not for the actual indexing, this is just to access the indexed data from Algolia. The crawler is configured via Algolia and is scheduled to run every 24 hours on production.

For staging, we still need the key to build and deploy without errors. I could set up a staging/test index for non-prod environments, but don't see the need for it yet.